manifests/repo.pp

```diff
@@ -0,0 +1,19 @@
+class sogo3::repo {
+  if $sogo3::repository_username and $sogo3::repository_password {
+    $location = "https://${sogo3::repository_username}:${sogo3::repository_password}@packages.inverse.ca/SOGo/release/3/debian/"
+  } else {
+    $location = 'https://packages.inverse.ca/SOGo/nightly/3/debian/'
```

Comment:
AFAICS, this only allow to use package for nightly or release from inverse.ca while Debian stretch does have a sogo package in official repo. Is there is any reason to prevent from using stable release from Debian repos ?

Reply:
That's the reason why a new `$use_custom_repo` parameter was added (default to `false`):
https://github.com/opus-codium/puppet-sogo3/pull/2/commits/1cb636fd555d55d0dea648770f36b85c944f4578#diff-60ae41fd0a31977447947f59940ee9a4R10

What is not currently managed is using a third party repository from the module, but it's out of scope AFAIAC.